ledger/participant-state/src/test/suite/scala/com/daml/ledger/participant/state/TimeModelTest.scala

```diff
@@ -12,7 +12,14 @@ class TimeModelTest extends WordSpec with Matchers {
   private val referenceTime = Instant.EPOCH
   private val epsilon = Duration.ofMillis(10L)
   private val timeModel =
-    TimeModel(Duration.ofSeconds(1L), Duration.ofSeconds(1L), Duration.ofSeconds(30L)).get
+    TimeModel(
+      minTransactionLatency = Duration.ofSeconds(1L),
+      maxClockSkew = Duration.ofSeconds(1L),
+      maxTtl = Duration.ofSeconds(30L),
+      avgTransactionLatency = Duration.ofSeconds(0L),
+      minSkew = Duration.ofSeconds(30L),
+      maxSkew = Duration.ofSeconds(30L),
```

Comment:
Don't think you need the `L` suffix; it should work fine without it.

Reply:
It looks like it's not necessary indeed. Why does it not trigger an implicit numeric widening error without the suffix?

Reply:
I _think_ (but I'm not sure) that it's because `30` is not an `Int`, is a numeric literal that can be converted to a concrete numeric type such as `Int` or `Long`. (I believe it can also be converted to `Float` or `Double` but Wart Remover complains about those.)